### ISSUE TITLE
[nftables] readline dependency mismatched

### ISSUE THREAD

**Issue by leoplo** [skycocker/chromebrew#4371]
The nft binary is currently looking for libreadline7, but Chromebrew is installing libreadline8. With a brand new install, trying to run nft command gives the following error:

```shell
nft: error while loading shared libraries: libreadline.so.7: cannot open shared object file: No such file or directory
```

## Versions and system information
chromebrew: 1.4.2

Model:
CHROMEOS_RELEASE_APPID={E7507B05-EC5C-437C-9878-E870E20111BC}
CHROMEOS_BOARD_APPID={E7507B05-EC5C-437C-9878-E870E20111BC}
CHROMEOS_CANARY_APPID={90F229CE-83E2-4FAF-8479-E368A34938B1}
DEVICETYPE=CHROMEBOOK
CHROMEOS_RELEASE_BOARD=lars
CHROMEOS_DEVSERVER=http://sd-137898.dedibox.fr:8080
GOOGLE_RELEASE=12607.89.2020_01_20_1922
CHROMEOS_RELEASE_BUILD_NUMBER=12607
CHROMEOS_RELEASE_BRANCH_NUMBER=89
CHROMEOS_RELEASE_CHROME_MILESTONE=79
CHROMEOS_RELEASE_PATCH_NUMBER=2020_01_20_1922
CHROMEOS_RELEASE_TRACK=developer-build
CHROMEOS_RELEASE_KEYSET=devkeys
CHROMEOS_RELEASE_NAME=Chromium OS
CHROMEOS_RELEASE_BUILD_TYPE=Developer Build - slapuser9
CHROMEOS_RELEASE_VERSION=12607.89.2020_01_20_1922
CHROMEOS_AUSERVER=http://sd-137898.dedibox.fr:8080/update
CHROMEOS_RELEASE_DESCRIPTION=12607.89.2020_01_20_1922 (Developer Build - slapuser9) developer-build lars

Architecture: x86_64

## Addtional information
Update nftables and libnftnl should do the trick.

**Comment by leoplo** [comment]
I opened a pull request for the update (https://github.com/skycocker/chromebrew/pull/4372) but I did not test on several architectures.

**Comment by uberhacker** [comment]
Fixed in #4372.
